src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/__tests__/StatusDropdown.spec.tsx

```diff
@@ -16,17 +18,10 @@ describe('StatusDropdown', () => {
     formikRenderer(<StatusDropdown name="status" />, {
       targets: { application: 'app' },
     });
-    expect(screen.queryByRole('button')).toBeInTheDocument();
-
-    await act(() => fireEvent.click(screen.getByRole('button')));
-
-    await waitFor(() => {
-      expect(screen.getByRole('menu')).toBeInTheDocument();
-      screen.getByText('Unresolved');
-    });
-    await act(() => fireEvent.click(screen.getByText('Unresolved')));
+    await user.click(screen.getByText('Select status of isssue'));
+    await user.click(screen.getByText('Unresolved'));
```

Comment:
why are you getting here by text and down bellow by id?

Reply:
User.click is an user event, the direct info for user is text options and user clicks the option to choose it. They do not care about the test id.
And for the Line 24, the option value/text is what we need to check. We do it to ensure the expected option is shown as the toggle value. So we cannot get by text.. here.